### DISCUSSION TITLE
SB 3600SE-10, e-Auto laden ohne SMA Batterieentladung

### DISCUSSION THREAD

**Discussion by wcosnc** [camueller/SmartApplianceEnabler#305] [Allgemein]
Hallo Axel,
Hallo Community,

ich bin bei meiner Recherche zur folgenden Problematik auf dein Projekt gestoßen und frage mich, ob das realisierbar ist?

Der SB 3600SE-10 hat einen eingebauten Li-Ionen Akku. Meine Wallbox verfügt über keine IoT Schnittstelle. Wenn ich das Auto anstecke, könnte ich den Ladestart über den Zwischenzähler (S0) erkennen. Ich möchte jetzt verhindern, dass der bereits geladene SMA Akku zur Aufladung des Fahrzeugs beiträgt, da das Auto ja ein zusätzlicher Speicher ist.  

Ist das möglich?

Danke und viele Grüße
Normann

**Comment by camueller** [comment]
Nein, Du brauchst eine steuerbare Wallbox.

**Comment by wcosnc** [comment]
Steuerbare Wallbox heißt dann: Überschussladen. Ich dachte man könnte über den Homemanager als Gerät einen "Akku" anbinden, sodass der WR nicht zusätzlich die interne Batterie nutzt. 

Vielen Dank für die Antwort.
